.github/workflows/release.yml

```diff
@@ -24,3 +26,68 @@ jobs:
           goarch: ${{ matrix.goarch }}
           ldflags: "-s -w -X main.version=${{ github.ref_name }}"
           binary_name: "gql-lint"
+
+  lipo:
+    name: Universal Mac binary
+    runs-on: macos-12
```

Comment:
We have some macos runners, I will open an issue to move this to those. I didn't want to mix things. Although I think it will be just seconds per release, and I wonder how many releases we will do...

Reply:
Probably very very very few (hopefully). If I were mac/infra I'd want to not use our own runners, just to have fewer dependencies on them...